shared-libs/transitions/src/lib/muting_utils.js

```diff
@@ -7,6 +8,10 @@ const infodoc = require('@medic/infodoc');
 infodoc.initLib(db.medic, db.sentinel);
 
 const BATCH_SIZE = 50;
+const OFFLINE = 'offline';
+const ONLINE = 'online';
```

Comment:
This will probably involve a lot of refactoring, but maybe we should be using "client" and "server" rather than "offline" and "online"? We should get the terminology right now, because we'll likely never get to change the database schema...

Reply:
Cool! I like that! Thanks!